bindgen/ir/TypeDef.h

```diff
@@ -23,6 +23,16 @@ class TypeDef : public TypeAndName, public Type {
 
     std::shared_ptr<Location> getLocation() const;
 
+    /**
+     * @return true if typedef references opaque type directly or through a
+     * chain of typedefs.
+     */
+    bool referencesOpaqueType() const;
+
+    bool usesOpaqueType() const override;
+
+    using TypeAndName::replaceType;
```

Comment:
Is this used?

Reply:
`TypeDef` inherits from `Type` and `TypeAndName`, so we need to specify what version of `replaceType` should be used.
A typedef may contain pointer to opaque type that should be replaced with pointer to `Byte`:
```c
typedef struct s *s;
```
```scala
type s = native.Ptr[Byte]
```